tests/test_validation_summary_logging.py

```diff
@@ -21,34 +21,34 @@ def should_log_event() -> None:
         task.lambda_handler(event, any_lambda_context())
 
         # Then
-        logger_mock.assert_any_call(expected_log)
+        logger_mock.assert_has_calls([expected_log_call])
 
 
 @patch("geostore.validation_summary.task.validation_results_model_with_meta")
 def should_log_failure_result(validation_results_model_mock: MagicMock) -> None:
     # Given
     event = {DATASET_ID_KEY: any_dataset_id(), VERSION_ID_KEY: any_dataset_version_id()}
-    expected_log = dumps({SUCCESS_KEY: False})
+    expected_log_call = call(LOG_MESSAGE_VALIDATION_SUCCESS, details=False)
     validation_results_model_mock.return_value.validation_outcome_index.count.return_value = 1
 
     with patch("geostore.validation_summary.task.LOGGER.debug") as logger_mock:
         # When
         task.lambda_handler(event, any_lambda_context())
 
         # Then
-        logger_mock.assert_any_call(expected_log)
+        logger_mock.assert_has_calls([expected_log_call])
 
 
 @patch("geostore.validation_summary.task.validation_results_model_with_meta")
 def should_log_success_result(validation_results_model_mock: MagicMock) -> None:
     # Given
     event = {DATASET_ID_KEY: any_dataset_id(), VERSION_ID_KEY: any_dataset_version_id()}
-    expected_log = dumps({SUCCESS_KEY: True})
+    expected_log_call = call(LOG_MESSAGE_VALIDATION_SUCCESS, details=True)
     validation_results_model_mock.return_value.validation_outcome_index.count.return_value = 0
 
     with patch("geostore.validation_summary.task.LOGGER.debug") as logger_mock:
         # When
         task.lambda_handler(event, any_lambda_context())
 
         # Then
-        logger_mock.assert_any_call(expected_log)
+        logger_mock.assert_has_calls([expected_log_call])
```

Comment:
As discussed, this can be simplified to `logger_mock.assert_any_call(LOG_MESSAGE_VALIDATION_SUCCESS, details=True)`.